src/Utils/normalization.jl

```diff
@@ -1,12 +1,3 @@
-# linear ivp in canonical form x' = Ax
-const LCF = LCS{N, AN} where {N, AN<:AbstractMatrix{N}}
-
-# affine ivp with constraints in canonical form x' = Ax + u, u ∈ U, x ∈ X
-const ACF = CLCCS{N, AN, IdentityMultiple{N}, XT, UT} where {N, AN<:AbstractMatrix{N}, XT<:LazySet{N}, UT<:AbstractInput}
-
-# type union of canonical forms
-const CF = Union{<:LCF, <:ACF}
-
 # accepted types of non-deterministic inputs (non-canonical form)
 const UNCF = Union{<:LazySet{N}, Vector{<:LazySet{N}}, <:AbstractInput} where {N}
```

Comment:
```suggestion
const UNCF{N} = Union{<:LazySet{N}, Vector{<:LazySet{N}}, <:AbstractInput}
```
Then add a type parameter in the respective methods. Otherwise we accept different numeric types and then it crashes during normalization.
(same for `XNCF`)

Reply:
good idea 👍